pkg/cmap/cmap_test.go

```
@@ -0,0 +1,213 @@
+
+		_, exists = m.Get("a")
+		assert.False(t, exists)
+	})
+}
+
+func randomIntn(n int) int {
+	b := make([]byte, 8)
+	if _, err := rand.Read(b); err != nil {
+		panic(err)
+	}
+	return int(binary.LittleEndian.Uint64(b) % uint64(n))
```

golangci-lint

[high] G115: integer overflow conversion int -> uint64

(gosec)